### PR TITLE
Rename pagedesigner

### PR DESCRIPTION
**Pull request by rbioteau**
Change plugin name
Update references to PageDesigner

### CODE REVIEW

File: bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/designer/core/repository/WebPageRepositoryStore.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public String getDisplayName() {
 
     @Override
     public Image getIcon() {
-        return Pics.getImage("page.png", PageDesignerPlugin.getDefault());
+        return Pics.getImage("page.png", UIDesignerPlugin.getDefault());
```
Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/designer/core/repository/WebPageRepositoryStore.java]: use a constants in PicsConstants for page.png

File: bundles/plugins/pom.xml [hunk 2]
```diff
@@ -43,6 +43,7 @@
 		<module>org.bonitasoft.studio.decision.core</module>
 		<module>org.bonitasoft.studio.decision.ui</module>
 		<module>org.bonitasoft.studio.dependencies</module>
+        <module>org.bonitasoft.studio.designer</module>
```
Comment (apupier) [inline bundles/plugins/pom.xml]: isn't an issue with indentation?

File: bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/designer/core/repository/WebPageRepositoryStore.java (hunk 1 above)
Comment (rbioteau) [inline bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/designer/core/repository/WebPageRepositoryStore.java]: I'm not a fan of using PicsConstants.
The constant should be decalred the closest possible place of where it used.
PicsConstants is relevant for images reused everywhere. This is not the case.

Comment (apupier) [inline bundles/plugins/org.bonitasoft.studio.pagedesigner/src/org/bonitasoft/studio/designer/core/repository/WebPageRepositoryStore.java]: ok, at least extract page.png as a constant in this class

File: bundles/plugins/pom.xml (hunk 2 above)
Comment (rbioteau) [inline bundles/plugins/pom.xml]: i do not see the problem in my eclipse... do you ?

**Pull request closed without merging**